tests/tests_models.py

```diff
@@ -0,0 +1,57 @@
+"""
+Defines tests for models in Catalog app
+"""
+
+from django.test import TestCase
+from blog.models import Page
+from django.conf import settings
+
+
+class ModelsTests(TestCase):
+
+    def setUp(self):
+        """
+        Defines testing data.
+        Mock config and tree pages with different types
+        """
+        settings.APP_BLOG_PAGE_TYPES = {
```

Comment:
@begor , кароч настройки в рантайме меняются с жуткими сайд-эффектами.
Моки не помогают абсолютно, в доках говорят, что это норм:
`Finally, avoid aliasing your settings as module-level constants as override_settings() won’t work on such values since they are only evaluated the first time the module is imported.`
https://docs.djangoproject.com/en/1.9/topics/testing/tools/#overriding-settings